download_clients/pivnet_client.go

```diff
@@ -180,7 +182,10 @@ func (p *pivnetClient) getLatestStemcell(dependencies []pivnet.ReleaseDependency
 	return stemcellSlug, stemcellVersion, nil
 }
 
-const errorForVersion = "versioning of stemcell dependency in unexpected format: \"major.minor\" or \"major\". the following version could not be parsed: %s"
+const (
+	errorForVersion = "versioning of stemcell dependency in unexpected format: \"major.minor\" or \"major\". the following version could not be parsed: %s"
+	userAgent       = "om-download-product"
```

Comment:
created a constant because AccessTokenService and pivnet.ClientConfig both need a userAgent